airbyte-integrations/bases/standard-source-test/src/main/java/io/airbyte/integrations/standardtest/source/AbstractSourceDatabaseTypeTest.java

```diff
@@ -87,29 +89,46 @@ protected void setupEnvironment(final TestDestinationEnv environment) throws Exc
    */
   protected abstract String getNameSpace();
 
+  /**
+   * Test the discover command. TODO (liren): This is a new unit test. Some existing databases may
+   * fail it, so it is turned off by default. It should be enabled for all databases eventually.
+   */
+  protected boolean testCatalog() {
+    return false;
```

Comment:
Is that expected to be committed or just accidentally forgotten?

Reply:
same comment

Reply:
This is expected. See the comment for the rationale. We need to check each database because we can turn it on by default.